DRAFT.md

```diff
@@ -804,9 +803,6 @@ explicit constexpr indirect(in_place_t, Us&&... us);
 
 14. _Postconditions_: `*this` is not valueless.
 
-15. _Throws_: Nothing unless `allocator_traits<allocator_type>::allocate` or
```

Comment:
The following paragraph numbers are not updated (16, 17, 18...), this should be updated to (15, 16, 17...)